packages/cloud-cognitive/src/components/PageHeader/PageHeader.js

```diff
@@ -93,6 +94,7 @@ export let PageHeader = React.forwardRef(
     hasCollapseHeaderToggle ??= deprecated_collapseHeaderToggleWanted;
     collapseTitle ??= deprecated_preCollapseTitleRow;
     disableBreadcrumbScroll ??= deprecated_preventBreadcrumbScroll;
+    const breadcrumbs = breadcrumbsIn ?? deprecated_breadcrumbItems;
```

Comment:
Is there a particular reason to handle this one slightly differently from the other deprecated props?

Reply:
I was experiencing infinite renders when trying to re-use `breadcrumbs` directly. It is possible I had issues elsewhere at the time.